### DISCUSSION TITLE
Organic Maps on Apple Mac

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2509] [Q&A]
I was surprised how useful the Desktop Version is. 
In my case its the Mac version. It imports my KML files very well. 

However I would like to **delete** some files from the Bookmarks but cannot see a Delete option. 
I need to tidy up my bookmarks.

Any suggestions welcome..

Thanks for a great product

**Comment by biodranik** [comment]
Find them on the disk ) I assume they are in ~/Library/Application Support/OMapsData/

**Reply by pwb999** [reply]
Sorry - cannot find any file or folder with OMapsData. Any other idea?

**Reply by biodranik** [reply]
Oh wait, did you install it from the App Store? Can you send a screenshot?

**Reply by pwb999** [reply]
Yes from the Apple Store.<img width="812" alt="Screenshot 2022-05-08 at 16 46 25" src="https://user-images.githubusercontent.com/58628197/167304059-42f23f3d-738b-46cc-92ae-80a643367bd8.png">

**Reply by biodranik** [reply]
Then search for it here: `/Users/YOUR_USER_NAME/Library/Containers/SOME_UNIQUE_NUMBER/Data/Documents/220415/`

**Comment by pwb999** [comment]
Excellent. Worked really well. 
Again many thanks for your help.
